app/code/core/Mage/Catalog/Model/Resource/Abstract.php

```diff
@@ -699,6 +699,43 @@ public function getAttributeRawValue($entityId, $attribute, $store)
         return $attributesData ? $attributesData : false;
     }
 
+    /**
+     * Retrieve attribute's label from DB.
+     *
+     * @param int $entityId
+     * @param int|string|array $attribute atrribute's ids or codes
+     * @param int|Mage_Core_Model_Store $store
+     * @return bool|string|array
+     */
+    public function getAttributeRawLabel($entityId, $attribute, $store)
```

Comment:
The name `getAttributeRawLabel()` doesn't seem correct to me, how about

```suggestion
    public function getAttributeText($entityId, $attribute, $store)
```

Reply:
I had a similar thought. How about `getAttributeRawText` to differentiate between `$product->getAttributeText('color');`. 

The "Raw" portion makes sense to me to match `getAttributeRawValue` since they both get data straight from the DB instead of going through the product model.